scripts/create_clone

```diff
@@ -259,24 +261,25 @@ $sysmod = "chmod 755 $caseroot/case.build";
 system ($sysmod); if ($? == -1) {die "$sysmod failed: $!\n";}
 
 # Create case.clean_build
-$sysmod = "cp $cimeroot/scripts/Tools/clean_build $caseroot/case.clean_build"; 
+$sysmod = "cp $cimeroot/scripts/Tools/case.clean_build $caseroot/case.clean_build";
 system($sysmod) == 0 or die "ERROR: $sysmod failed: $?\n";
-    
+
 # Create case.submit
-$sysmod = "cp  $cimeroot/scripts/Tools/case.submit $caseroot/case.submit"; 
+$sysmod = "cp  $cimeroot/scripts/Tools/case.submit $caseroot/case.submit";
 system($sysmod) == 0 or die "ERROR: $sysmod failed: $?\n";
 
 
 #-----------------------------------------------------------------------------------------------
 # Create batch script
-$sysmod = "./case.setup"; 
+$ENV{PYTHONPATH} = "$cimeroot/scripts/Tools:$cimeroot/utils/python";
```

Comment:
I think that this is a temporary measure until we get rid of the perl.   I do have some concern that the files created from templates such as case.run will need something like this since they are not linked from the source tree as the others are.